app/assignment/directives/modals/commentModal/commentModal.html

```diff
@@ -4,7 +4,10 @@
       No Instructor comments.
     </div>
 
-    <div class="comment-modal__comment" ng-show="instructorComment.length > 0">{{ instructorComment }}</div>
+    <div class="comment-modal__comment" ng-repeat="comment in instructorComment | orderBy: '-lastModifiedOn'">
```

Comment:
Whats the - do?
Is that a reverse order flag? :D

Reply:
Yup, sorts item in descending order. Seems to work fine in this simple use case.